CMakeLists.txt

```diff
@@ -91,6 +91,7 @@ option(GINKGO_DPCPP_SINGLE_MODE "Do not compile double kernels for the DPC++ bac
 option(GINKGO_INSTALL_RPATH "Set the RPATH when installing its libraries." ON)
 option(GINKGO_INSTALL_RPATH_ORIGIN "Add $ORIGIN (Linux) or @loader_path (MacOS) to the installation RPATH." ON)
 option(GINKGO_INSTALL_RPATH_DEPENDENCIES "Add dependencies to the installation RPATH." OFF)
+option(GINKGO_INSTALL_GDB_PRETTY_PRINTER "Install the GDB pretty-printer for auto-loading." OFF)
```

Comment:
To reduce the number of CMake options we would need to manage, maybe just bundle the pretty-printer with `GINKGO_DEVEL_TOOLS` ?

Reply:
Those are two different things - one of them is used for development purposes (where you usually don't install your code), and the other one is for application debugging purposes (on an installed Ginkgo). But it definitely makes sense to copy the pretty-printer next to the compiled libginkgo.so for debugging purposes.

Reply:
Another option to approach this (probably also nice for packagers) is to use `install(... COMPONENT Development)`, this also makes sense for #1144

Reply:
Yes, I think that is a good idea!